src/core/LegacyMeganav/SignOutLink.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { MouseEvent, MouseEventHandler, ReactNode, useRef } from "react";
```

Comment:
Just bear in mind that this moved file will cause breaking changes in other repos, as this component is used outside of a meganav context

Reply:
thanks @jamiehenson for this, lucky enough we have moved this to the website so we wont call this component again in `ably-ui`